app/src/main/res/drawable/cursor_full_active.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
nit: it'd be easier to review this patch if you didn't add these files in this commit. I had to skim ahead to ensure they're used appropriately, unchanged, etc. (or removed in the case of that last file).

But generally, good job splitting this up into separate commits! :) Do you know about interactive rebase? `git rebase -i <ancestor-commit>`. This is how you can move commits, squash them (e.g. like moving the unnecessary commit file in here), etc.